locales/en/common.json

```diff
@@ -10,7 +22,12 @@
       "notes": "Notes",
       "others_flagged": "Others who flagged",
       "title": "Flag"
-    }
+    },
+    "odk": {
+      "title": "Login to ROBUST Web platform as webuser",
+      "qrCode": "QR Code"
```

Comment:
can you please sort these translation files?
suggestion: `jsonsort /path/to/locales/directory`

Reply:
fixed